### DISCUSSION TITLE
Add option to show beatmap statistics in top left beatmap card

### DISCUSSION THREAD

**Discussion by jasonsbrar** [ppy/osu#17855] [Ideas] [answered]
It would be nice to have the ability to be able to see the leaderboard and the beatmap statistics (ar, od, cs, etc.) in the same view without having to switch back and forth.
![songselect](https://user-images.githubusercontent.com/14091988/163735694-0b20a607-81c1-4621-bca2-3e503b50ff71.png)

**Comment by integerrr** [comment, marked as answer]
Song select is getting an eventual redesign which should address this:
https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=1%3A1276
